examples/offboard_position_velocity_ned.py

```diff
@@ -42,7 +42,7 @@ async def print_z_velocity(drone):
         async for odom in drone.telemetry.position_velocity_ned():
             print(f"{odom.velocity.north_m_s} {odom.velocity.down_m_s}")
 
-    tasks = {asyncio.create_task(print_z_velocity(drone))}
+    _tasks = {asyncio.create_task(print_z_velocity(drone))}
```

Comment:
It is unusual to assemble these tasks but not await them.  Are we certain that they will be run at some point?